### ISSUE TITLE
More accessible invalid field indications

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#1346]
### Is your feature request related to a problem?

We have colour indications of whether certain input fields are invalid (such as empty required field, incorrect format in input type=email, etc.) - however this is not accessible for those with red colour blindness. We should also provide iconic indicators where fields are invalid. See recommendation here: https://developer.mozilla.org/en-US/docs/Web/CSS/:invalid#Accessibility_concerns

### What is the feature?

Add iconic indications of invalid fields in the core admin themes.

**Comment by philwareham** [comment]
I've just done some tests for this and modern browsers natively do a good job of announcing invalid fields - such as when a required field is empty or input type=email does not contain the proper email format. So I'm happy to close this issue without further work.
